utils.js

```diff
@@ -216,7 +216,7 @@ const _currentDesktopsMatches = {};
 //
 // This function is a copy of:
 // https://github.com/endlessm/gnome-shell/blob/master/js/misc/desktop.js
-function desktopIs(name, maxVersion = '3.38') {
+function desktopIs(name, maxVersion = '3.38.0') {
```

Comment:
Shouldn't a more appropriate default be `config.PACKAGE_VERSION` here?

Reply:
Yes, sure, good idea. That will fix future problems.